python/setup.py

```diff
@@ -46,6 +47,12 @@
 exec(compile(open(libinfo_py, "rb").read(), libinfo_py, 'exec'), libinfo, libinfo)
 
 LIB_PATHS = libinfo['find_lib_path']()
+LIB_PATHS += glob.glob("/".join(LIB_PATHS[0].split("/")[0:-1]) + "/*.so*")
+symlinks = []
+for src in set(LIB_PATHS):
+  symlinks.append('mxnet/' + src.split('/')[-1])
+  os.symlink(src, symlinks[-1])
```

Comment:
It's not immediately obvious what this code is trying to accomplish.  A comment or two might be useful here.